### ISSUE TITLE
New theme proposal for the roadmap

### ISSUE THREAD

**Issue by RyuLindow** [umbraco/Umbraco.Uno.Issues#30]
### Description:

I would like to propose a new theme, as we have that on the roadmap.

The theme will be a portfolio directed towards photographers, graphic designers, freelance devs, etc.

It would also be great if widgets like Gallery and Image could get an overlay option like the Hero one does but extended, so you can add an "on hover" option.

This is a good example - http://forbescreative.co.uk/ - of what I'm trying to describe.

Basically, a new theme that is more freelancer oriented.

**Comment by nul800sebastiaan** [comment]
Thanks @RyuLindow we'll consider it when we start thinking about new themes to implement! 👍
